generators/generator-constants.js

```diff
@@ -22,6 +22,7 @@ const DOCKER_CONSUL_CONFIG_LOADER = 'jhipster/consul-config-loader:v0.2.1';
 const DOCKER_PROMETHEUS = 'prom/prometheus:v1.4.1';
 const DOCKER_PROMETHEUS_ALERTMANAGER = 'prom/alertmanager:v0.5.1';
 const DOCKER_GRAFANA = 'grafana/grafana:4.0.2';
+const DOCKER_JENKINS = 'jenkins:latest';
```

Comment:
@pascalgrimaud I'm really against it as being in an alpine environment is annoying for debugging and It will create a lot of problems. Let's use the default image.